src/runtime-rs/crates/hypervisor/src/remote/mod.rs

```diff
@@ -48,9 +48,18 @@ impl Hypervisor for Remote {
         id: &str,
         netns: Option<String>,
         annotations: &HashMap<String, String>,
+        // NOTE: SELinux label support for remote is currently not available due to developer reasons.
```

Comment:
What are "developer reasons"? I think we can remove this comment and keep the following `warn!`.